sc2ts/info.py

```diff
@@ -615,12 +611,7 @@ def _preprocess_mutations(self, show_progress):
         )
         ancestral_state = tables.sites.ancestral_state.view("S1").astype(str)
         del tables
-        inherited_state = ancestral_state[ts.mutations_site]
```

Comment:
Can delete the stuff above pulling out the table entirely - that's just to get the arrays